modules/apps/transfer/types/transfer_authorization.go

```diff
@@ -1,17 +1,23 @@
 package types
 
 import (
+	"math/big"
+
 	errorsmod "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/authz"
-
 	ibcerrors "github.com/cosmos/ibc-go/v7/internal/errors"
 	channeltypes "github.com/cosmos/ibc-go/v7/modules/core/04-channel/types"
 	host "github.com/cosmos/ibc-go/v7/modules/core/24-host"
 )
 
 var _ authz.Authorization = (*TransferAuthorization)(nil)
 
+// MaxUint256 is the maximum value for a 256 bit unsigned integer used in EVM compatible chains.
+var (
+	MaxUint256 = new(big.Int).Sub(new(big.Int).Lsh(big.NewInt(1), 256), big.NewInt(1))
+)
+
```

Comment:
remove var, it's not necessary for anything but the check

Reply:
We could make it unexported, but atm its leveraged in tests. I like the idea of having this as a package level variable so it can be evaluated once on pkg init.